src/VeresModel.cc

```diff
@@ -92,6 +92,25 @@ std::vector<double> VeresModel::gradient(std::vector<double> const& params) cons
     return gradChiSq;
 }
 
+std::shared_ptr<ImageF> VeresModel::computeModelImage(std::vector<double> const& params) const {
+    double xc = params[0];     // Centroid x
+    double yc = params[1];     // Centroid y
+    double flux = params[2];   // Flux
+    double length = params[3]; // Trail length
+    double theta = params[4];  // Angle from +x-axis
+
+    // Loop is adapted from lsst::afw::detection::Psf::computeKernelImage()
```

Comment:
more like GaussianPsf::doComputeKernelImage?

Reply:
Yes, that is more specific, thanks.